README.md

```diff
@@ -4,110 +4,36 @@ Whom can we blame for using all our disk space?  Find out here.
 
 Developed by Ryan Eckbo, Tashrif Billah, and Isaiah Norton
 
-Table of Contents
-=================
-
-   * [diskusage-logging](#diskusage-logging)
-   * [Installation](#installation)
-      * [1. Install <em>R</em> in Linux](#1-install-r-in-linux)
-      * [2. Check installation (/usr/bin/R)](#2-check-installation-usrbinr)
-      * [3. Version](#3-version)
-      * [4. Prerequisite libraries](#4-prerequisite-libraries)
-   * [Configuration](#configuration)
-   * [Running](#running)
-   * [Output](#output)
-   * [Report](#report)
-      * [1. Make report](#1-make-report)
-      * [2. Mail report](#2-mail-report)
-      * [3. Categorized spreadsheet](#3-categorized-spreadsheet)   
-   * [cron job scheduling](#cron-job-scheduling)
-      * [1. <em>cron.d</em> directory](#1-crond-directory)
-      * [2. User crontab](#2-user-crontab)
-   * [Issues](#issues)
-
-
-Table of Contents Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
-
 
 # Installation
 
-## 1. Install *R* in Linux
-
-    sudo yum install R
-    
-## 2. Check installation (`/usr/bin/R`)
-
-    which R
-    R --help
-    R # should initiate R console
-
-## 3. Version
-
-Current *diskusage-logging* is on the following version of R:
-    
-    > version
-    
-    platform       x86_64-redhat-linux-gnu
-    arch           x86_64
-    os             linux-gnu
-    system         x86_64, linux-gnu
-    status
-    major          3
-    minor          5.2
-    year           2018
-    month          12
-    day            20
-    svn rev        75870
-    language       R
-    version.string R version 3.5.2 (2018-12-20)
-    nickname       Eggshell Igloo
-
-    > sessionInfo()
-    
-    sessionInfo()
-    R version 3.5.2 (2018-12-20)
-    Platform: x86_64-redhat-linux-gnu (64-bit)
-    Running under: CentOS release 6.8 (Final)
-
-    Matrix products: default
-    BLAS: /usr/lib64/R/lib/libRblas.so
-    LAPACK: /usr/lib64/R/lib/libRlapack.so
-    ...
-    ...
-
-    
-## 4. Prerequisite libraries
-The following libraries are installed in `/usr/lib64/R/library/` (you can check version info
-with `packageVersion("knitr")`:
-
-
-    knitr      = 1.22
-    ggplot2    = 3.1.1
-    xtable     = 1.8.3
-    magrittr   = 1.5
-    knitr      = 1.22
-    bit64      = 0.9.7
-    lubridate  = 1.6.0
-    data.table = 1.11.5
-    
-
-* NOTE: the current version of `lubridate` and `data.table` in CRAN, requires 
-a newer compiler than is available on CentOS 6. If required, you can install an older version of 
-them directly from source:
+## 1. Create virtual environment (optional)
```

Comment:
Please remove this point and start from ## 2. below. Users are expected to know how to install Python and activate environment.

Reply:
I removed the subsection and integrated it with the title